lib/Mojo/Util.pm

```diff
@@ -67,6 +67,7 @@ my $ENTITY_RE = qr/&(?:\#((?:[0-9]{1,7}|x[0-9a-fA-F]{1,6}));|(\w+[;=]?))/;
 # Encoding and pattern cache
 my (%ENCODING, %PATTERN);
 
+# re-exporting BaseUtil functions for backward compatibility
```

Comment:
I would consider these re-exports a permanent functionality matter, not backwards compatibility - I don't see a reason to ever remove these from Mojo::Util. This comment isn't really needed